scripts/upload_results_to_oep.py

```diff
@@ -0,0 +1,143 @@
+# coding: utf-8
+r"""
+Inputs
+-------
+
+Outputs
+---------
+
+Description
+-------------
+Creates tables on the OpenEnergyPlatform (OEP) based on related oemetadata. Uploads results data
+in oemof_b3-format to the OEP.
+
+The script can delete existing tables on the OEP that have been uploaded with an affiliated OEP user
+account.
+
+The oemetadata format is a standardised json file format and is required for all data uploaded to
+the OEP. It includes the data model, the used data types, and general information about the data
+context. Tables in sqlalchemy are created based on the information in the oemetadata.
+"""
+import os
+import sys
+
+import pandas as pd
+from unittest.mock import Mock
+
+# try:
+#     from oem2orm import oep_oedialect_oem2orm as oem2orm
+# except ImportError:
+#     raise ImportError("Need to install oem2orm to upload results to OEP.")
+
+from oemof_b3.config import config
+
+oem2orm = Mock()
+
+SCHEMA = "model_draft"
+
+
+if __name__ == "__main__":
+    filepath = sys.argv[1]
+    metadata_path = sys.argv[2]
+    logfile = sys.argv[3]
+
+    # set up the logger
+    scenario = "scenario"  # TODO: Derive scenario from filepath
+    logger = config.add_snake_logger(logfile, "upload_results_to_oep")
+
+    # find data to upload
+    list_filenames = os.listdir(filepath)
+
+    logger.info("These files will be uploaded: " + ", ".join(list_filenames))
+
+    # Setting up the oem2orm logger
+    # If you want to see detailed runtime information on oem2orm functions or if errors occur,
+    # you can activate the logger with this simple setup function.
+    # oem2orm.setup_logger()
```

Comment:
Can we drop this, @chrwm?

Reply:
Done in [1448923](https://github.com/rl-institut/oemof-B3/pull/259/commits/1448923cb2765caaa03bc03e8a2ddcf03e02c09f)